test/test_registers.py

```diff
@@ -1,8 +1,17 @@
+import pytest
+
 from opensquirrel import Circuit
 from opensquirrel.passes.decomposer import McKayDecomposer
+from opensquirrel.passes.merger.general_merger import Merger
+from opensquirrel.passes.merger.single_qubit_gates_merger import SingleQubitGatesMerger
+
+
+@pytest.fixture(name="merger")
+def merger() -> SingleQubitGatesMerger:
+    return SingleQubitGatesMerger()
```

Comment:
Here the fixture should not be used, because this is test module does not focus specifically on the SingleQubitGateMerger.
```suggestion
```